Modules/xASL_module_ASL.m

```diff
@@ -194,6 +193,16 @@
     fprintf('%.2f\n', round(x.Q.uniqueEchoTimes(end),4));
 end
 
+% If multi-TE data is detected, we switch on multi-TE quantification by default. Unless this has been deactived in dataPar.json
```

Comment:
Weird. bMultiTE has no function. Detecting multiple echo times is just uniqueEchoTimes>1 right?
We also don't have a bMultiLD ?

Reply:
As above -> You insisted on having this for clarity.

Reply:
Yes, probably because this meant a different behavior of ExploreASL. Now we are rewriting this into the same behavior right (at least control ASL subtraction, Hadamard decoding, etc, will run the same code and bMultiTE is not used anymore, only bMultiTEQuantification is used

Reply:
OK. I can remove bMultiTE, but you need to realize that we are rewriting the code there and back with no real reason. I hope that in one year you will not argue that we absolutely need to have bMultiTE.